<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cyclemanagement</class>
 <widget class="QMainWindow" name="cyclemanagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion du Cycle d'Ingénieur</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image: url(:/assets/fst.jpeg);
    background-repeat: no-repeat;
    background-position: center;
}
QWidget#centralwidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
                               stop:0 rgba(0, 123, 255, 0.35),
                               stop:1 rgba(0, 123, 255, 0.3));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="sidebarWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#sidebarWidget {
    background-color: rgba(41, 128, 185, 0.9);
    border-radius: 15px;
    margin: 10px;
}
QComboBox {
    background-color: white;
    border-radius: 5px;
    padding: 5px;
    margin: 10px;
    font-size: 14px;
    font-family: 'Segoe UI', Arial;
}
QPushButton {
    background-color: rgba(255, 255, 255, 0.1);
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px;
    margin: 5px 10px;
    text-align: left;
    font-size: 14px;
    font-family: 'Segoe UI', Arial;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.2);
}
QPushButton:pressed {
    background-color: rgba(255, 255, 255, 0.3);
}
QPushButton#logoutButton {
    background-color: rgba(231, 76, 60, 0.8);
    margin-top: 20px;
    margin-bottom: 10px;
    font-weight: bold;
}
QPushButton#logoutButton:hover {
    background-color: rgba(231, 76, 60, 1.0);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="levelComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="studentsButton">
         <property name="text">
          <string>Étudiants</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="professorsButton">
         <property name="text">
          <string>Professeurs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scheduleButton">
         <property name="text">
          <string>Emploi du temps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="internshipsButton">
         <property name="text">
          <string>Stages PFE/PFA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="requestsButton">
         <property name="text">
          <string>Les demandes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutButton">
         <property name="text">
          <string>Déconnexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="styleSheet">
       <string notr="true">QStackedWidget {
    background-color: rgba(255, 255, 255, 0.9);
    border-radius: 15px;
    margin: 10px;
}
QTableWidget {
    background-color: transparent;
    border: none;
    gridline-color: #bdc3c7;
    font-family: 'Segoe UI', Arial;
    font-size: 13px;
}
QHeaderView::section {
    background-color: #2980b9;
    color: white;
    padding: 5px;
    border: none;
    font-weight: bold;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
